makefiles/arch/cortexm.inc.mk

```diff
@@ -16,7 +16,16 @@ export CFLAGS += $(CFLAGS_CPU) $(CFLAGS_LINK) $(CFLAGS_DBG) $(CFLAGS_OPT)
 
 export ASFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG)
 export LINKFLAGS += -L$(RIOTCPU)/$(CPU)/ldscripts -L$(RIOTCPU)/cortexm_common/ldscripts
-export LINKER_SCRIPT ?= $(CPU_MODEL).ld
+
+ifneq (,$(filter stm32%,$(CPU_FAM)))
```

Comment:
this seems wrong. There should be no multiplexing for different families in the shared cortex code tree. Instead, this should be handled by the families makefiles!

Reply:
To be more precise: leave `export LINKER_SCRIPT ?= $(CPU_MODEL).ld` in this file. But move the other two options to the corresponding cpu family makefile.includes. As we use `?=` for assignment here, and the family makefiles are passed before this one, the more specific values prevail.

Reply:
Yes! I prefer your suggestion, I was looking where to put it but didn't think about `?=` (it always happen...). Will fix.